mmdet/core/post_processing/matrix_nms.py

```diff
@@ -37,7 +37,7 @@ def mask_matrix_nms(masks,
             - scores (Tensor): Updated scores, has shape (n,).
             - labels (Tensor): Remained labels, has shape (n,).
             - masks (Tensor): Remained masks, has shape (n, w, h).
-            - keep_inds (Tensor): The indexs number of
+            - keep_inds (Tensor): The indices number of
```

Comment:
indexes or indices?

Reply:
Indices and indexes are all right.
`indexes` is often used in North American countries, and other regions often use `indices`.

Reply:
It is suggested to use indexes due to we used `indexes` below. Or we need to unify